portfolio/src/main/webapp/script.js

```diff
@@ -2,20 +2,32 @@
 // This function generates a random quote to display, also making sure that the previously displayed random image is not the same as the current random image.
 function randomizeQuote() {
 
+  const range= 4;    // This constant variable keeps track of the number of quotes I have. 
+
   // ImageNumber has the Image index of the quotes. This generates a number between [1-4]
```

Comment:
nit: in the comment you can change the range to be [1-range]. Generally we don't want to have 4 anywhere.